vm/systemSmartContracts/validator.go

```diff
@@ -907,15 +940,22 @@ func (v *validatorSC) unStake(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 	return vmcommon.Ok
 }
 
-func (v *validatorSC) unStakeNodeOneNodeFromStakingSC(blsKey []byte, rewardAddress []byte) vmcommon.ReturnCode {
-	vmOutput, errExec := v.executeOnStakingSC([]byte("unStake@" + hex.EncodeToString(blsKey) + "@" + hex.EncodeToString(rewardAddress)))
-	if errExec != nil {
+func (v *validatorSC) unStakeNodes(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if !v.flagEnableTopUp.IsSet() {
```

Comment:
flagEnableTopUp because it will be enabled with the Topup?

Reply:
yes. new functions on validatorSC will be enabled with topup

Reply:
ok